### DISCUSSION TITLE
Help with timeformat and first zero

### DISCUSSION THREAD

**Discussion by Francaisedolon** [vydimitrov/react-countdown-circle-timer#251] [Q&A] [answered]
Hello...
I use the timer for a clock countdown.
But the problem is missing the first zero when umber only 1 length.

Example what I need
04:08:59

Example what the timer make:
4:8:59

Have you a solution for me?

Thank you

Best regards

**Comment by vydimitrov** [comment, marked as answer]
You may want to read this to see how you can format the time - https://javascript.info/date. That is up to you, as the timer just gives you the remaining time, so you can format it as you want.

**Comment by Francaisedolon** [comment]
Ok thank you.
I found a solution
